tests/yam/agama/boot_from_hd_after_agama.pm

```diff
@@ -0,0 +1,19 @@
+## Copyright 2024 SUSE LLC
```

Comment:
we could call the yaml and this test, `agama_boot_from_hdd`. I think we should consider to prefix all in the future with agama to avoid collision, specially for this modules of booting where Perl might find a similar name before ours if having the same name.